replica/database.cc

```diff
@@ -1028,12 +1028,10 @@ future<> database::add_column_family(keyspace& ks, schema_ptr schema, column_fam
     ks.add_or_update_column_family(schema);
     cf->start();
     schema->registry_entry()->set_table(cf->weak_from_this());
-    _tables_metadata._column_families.emplace(uuid, std::move(cf));
-    _tables_metadata._ks_cf_to_uuid.emplace(std::move(kscf), uuid);
+    co_await _tables_metadata.add_cf(uuid, std::move(kscf), std::move(cf));
```

Comment:
we could pass the schema_ptr for uuid, ks_name, and cf_name.
I think the class is exposing too much of its internals by exposing e.g. ks_cf_t in the class public interface.